apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/components/CareerTimelineAndRecruitment.tsx

```diff
@@ -61,127 +63,154 @@ const CareerTimelineAndRecruitment = ({
   const paths = useRoutes();
   const hasCareerTimelineItems = !!experiences?.length;
 
+  let crumbs = [
+    {
+      label: intl.formatMessage(titles.careerTimelineAndRecruitment),
+      url: paths.careerTimelineAndRecruitment(applicantId),
+    },
+  ];
+  if (!pool) {
```

Comment:
I'm confused by this, I can see it was referenced in this file before but what is it for?

From what I can tell about what is going on with it 
The prop comes from pool being passed in from here in `CareerTimelineAndRecruitmentApi`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/8cdf7ff4-5f9d-4ed5-8bcb-e25ec11f7785)

Which is rendered if `applicationId` is true in `ApiOrContent`

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/52b49b04-c0a1-4af0-a2bb-e16016c4ea8d)

Which comes from reading the URL in CareerTimelineAndRecruitmentPage

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/901fac7c-d227-4541-8125-533f05aae8f3)

Reply:
If you try and navigate like that and get the `!pool` to be false you get this page which seems... wrong? It combines missing skills with the timeline

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/12ba3959-2521-4743-baf9-f5808ef0e15e)

Reply:
I was a little confused by it as well 😅 I just added it since it already existed. Do you think we could safely remove that part as well?

Reply:
I'm not even sure what it is trying to accomplish, I was trying it out to test out breadcrumbs thoroughly but got stumped trying to find out why it is even there. If you try and go to that page to check breadcrumbs it doesn't even look right

Reply:
Oh, is this left over from when we could navigate to this page from the application process? 🤔